src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -117,16 +117,6 @@ public IHttpActionResult GetHostStatus()
                     status.Errors.Add(Utility.FlattenException(lastError));
                 }
 
-                var parameters = Request.GetQueryParameterDictionary();
```

Comment:
Verified with Joseph that this "load status" endpoint was never consumed by him, so we're safe to remove. It will be replaced in the future with the runtime scale work.